server/src/test/java/org/elasticsearch/index/codec/bloomfilter/ES93BloomFilterStoredFieldsFormatTests.java

```diff
@@ -132,6 +114,31 @@ int getBloomFilterSizeInBits() {
                     };
                 }
             });
+            var bloomFilterSizeInKb = atLeast(2);
+            conf.setCodec(
```

Comment:
We set the codec two times in this test, here and at line 93. I guess the later one which uses `TestCodec` it the one we want to keep.